### DISCUSSION TITLE
请问如何获得命令发出者的qq号？

### DISCUSSION THREAD

**Discussion by TDK1969** [nonebot/discussions#117] [问答] [answered]
环境：
nonebot2
go-cqhttp

**Comment by yanyongyu** [comment, marked as answer]
根据事件内容获取，如 `event.user_id`

具体请参考onebot文档 https://onebot.adapters.nonebot.dev/docs/api/v11/event#MessageEvent

**Reply by TDK1969** [reply]
已解决，感谢
